react/features/calendar-sync/actions.js

```diff
@@ -242,3 +243,42 @@ export function updateProfile(calendarType: string): Function {
             });
     };
 }
+
+/**
+ * Updates calendar event by generating new invite URL and editing the event
+ * adding some descriptive text and location.
+ *
+ * @param {string} id - The event id.
+ * @param {string} calendarId - The id of the calendar to use.
+ * @returns {Function}
+ */
+export function updateCalendarEvent(id: string, calendarId: string): Function {
+    return (dispatch: Dispatch<*>, getState: Function) => {
+
+        const { integrationType } = getState()['features/calendar-sync'];
+        const integration = _getCalendarIntegration(integrationType);
+
+        if (!integration) {
+            return Promise.reject('No integration found');
+        }
+
+        const { locationURL } = getState()['features/base/connection'];
+        const newRoomName = generateRoomWithoutSeparator();
+        let href = locationURL.href;
```

Comment:
Double checking: this will only work on the welcome because otherwise in a meeting the href will include a meeting and that's okay?

Reply:
Hum, yeah, I wonder should we handle this. Currently, the only place to edit will be welcomepage ... WDYT?

Reply:
We'll cross that bridge when we get there.